test/components/Home.vue

```diff
@@ -58,6 +62,11 @@ export default {
   components: {CredentialCard, CredentialCardDetail, CredentialCardList, Logo},
   data() {
     return {
+      profile: {
```

Comment:
This should be removed. Again, specific to a product and not needed in this component. It should just receive the color.